src/TimeTrack.cpp

```diff
@@ -254,7 +254,7 @@ void TimeTrack::Draw(wxDC & dc, const wxRect & r, double h, double pps)
                             //
                             // LL:  It's because the ruler only Invalidate()s when the new value is different
                             //      than the current value.
-   mRuler->SetFlip(GetHeight() > 75 ? true : true); // MB: so why don't we just call Invalidate()? :)
+   mRuler->SetFlip(true);   // MB: so why don't we just call Invalidate()? :)
```

Comment:
Seams I ran into the same mistake… But I was able to fix it by reassigning mValue later. The fix is in my repo, but won't sync up here, because the pull request is closed now.